reactor-core/src/test/java/reactor/core/publisher/MonoUsingTest.java

```diff
@@ -59,15 +132,12 @@ public void resourceCleanupNull() {
 		});
 	}
 
-	@Test
-	public void normal() {
+	@ParameterizedTestWithName
```

Comment:
Currently different test cases are indistinguishable when they fail. The suggestion for `CleanupCase` wrapper class can be used to add a `name` field and then implement a `toString` method that uses the `name`. This way we can see exactly which scenario failed. Please consult e.g. `ContextLossDetectionTest` to see how that works.

Reply:
![image](https://github.com/reactor/reactor-core/assets/34854527/3e26685f-d14d-4bd6-82b9-c77ede89e0bc)

✅ updated! override `toString`, now I can distinguish each test by `name` 👍

Reply:
> Please consult e.g. ContextLossDetectionTest to see how that works.

Thanks for helpful guide! it saved my time a lot.